ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -61,7 +61,7 @@ resources:
       memory: 200Mi
 tests:
 - as: baremetalds-ipi-ovn-ipv4-fips-f28
-  cron: 7 17 13 * *
```

Comment:
Hmm, seem like you did not update your local `generate-cron-entry.sh` tools, mine did not get change.
```
[jianlinliu@jialiu-laptop release]$ generate-cron-entry.sh baremetalds-ipi-ovn-ipv4-fips-f28 openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml
cron: 7 17 13 * *
```

Reply:
Hmm, I ran update-cron-entries.py and all were updated.

```
yangyang:tools/ (eus-nutanix✗) $ ./update-cron-entries.py ../openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml
Updating ../openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml
updating test job - baremetalds-ipi-ovn-ipv4-fips-f28
55 4 24 * *
updating test job - gcp-ipi-workload-identity-rt-fips-f28
11 17 9 * *
updating test job - ibmcloud-ipi-f28
21 13 5 * *
updating test job - nutanix-ipi-proxy-fips-f28
19 1 5 * *
updating test job - vsphere-ipi-f28
22 4 2 * *
```

Reply:
update-cron-entries.py is calling generate-cron-entry.sh to set the cron entires.

Reply:
btw, can you switch to `../` folder to run `./update-cron-entries.py openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml`

Reply:
right, that's the root cause. Updated!